benchmarks/imagenet/resnet50/simclr.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Tuple
+
+from pytorch_lightning import LightningModule
+from torch import Tensor
+from torch.nn import Identity
+from torchvision.models import resnet50
+
+from lightly.loss.ntx_ent_loss import NTXentLoss
+from lightly.models.modules import SimCLRProjectionHead
+from lightly.models.utils import get_weight_decay_parameters
+from lightly.transforms import SimCLRTransform
+from lightly.utils.benchmarking import OnlineLinearClassifier
+from lightly.utils.lars import LARS
+from lightly.utils.scheduler import CosineWarmupScheduler
+
+
+class SimCLR(LightningModule):
+    def __init__(self, batch_size: int, num_classes: int) -> None:
+        super().__init__()
+        self.save_hyperparameters()
+        self.batch_size = batch_size
+
+        resnet = resnet50()
+        resnet.fc = Identity()  # Ignore classification head
```

Comment:
I used this instead of the `self.backbone = nn.Sequential(*list(resnet.children())[:-1])` trick as this should make it easier to load the saved checkpoints. And is IMHO also cleaner.